deployment/charts/universalnode/helmfile.yaml

```diff
@@ -7,3 +7,7 @@ releases:
       values:
         - name: {{ .Values.name }}
         - imageTag: {{ requiredEnv "TAG" }}
+        - contractAddress: {{ .Values.name  }}
```

Comment:
Similarly to the parent helmfile, use `- {{ toYaml .Values | nindent 8 }}`, but this time you can replace all the parameters (also name and imageTag) as they are already coming from the parent helmfile's map